tests/sentry/tasks/test_on_demand_metrics.py

```diff
@@ -603,3 +603,32 @@ def assert_on_demand_model(
         return
 
     assert model.spec_hashes == expected_hashes[model.spec_version]  # Still include hashes
+
+
+@django_db_all
+def test_support_aggregates(project: Project) -> None:
+    query = "transaction.duration:>100"
+    columns = ["apdex(300)", "transaction"]
```

Comment:
Can we explicitly check that `transaction` is in the group-by of the spec here?

Reply:
See `assert metric_specs[-1][1]` in the section below.